specification/databox/resource-manager/Microsoft.DataBox/stable/2019-09-01/databox.json

```diff
@@ -2120,70 +2692,242 @@
-          "description": "The list of key value pairs that describe the resource. These tags can be used in viewing and grouping this resource (across resource groups).",
-          "type": "object",
-          "additionalProperties": {
-            "type": "string"
-          }
-        },
-        "sku": {
-          "$ref": "#/definitions/Sku",
-          "description": "The sku type."
+      "allOf": [
+        {
+          "$ref": "#/definitions/ValidationInputRequest"
         }
-      },
-      "x-ms-azure-resource": true
-    },    
-    "ShareCredentialDetails": {
-      "description": "Credential details of the shares in account.",
-      "type": "object",
+      ],
       "properties": {
-        "shareName": {
-          "description": "Name of the share.",
-          "type": "string",
-          "readOnly": true
+        "preference": {
+          "description":"Preference requested with respect to transport type and data center",
+          "$ref": "#/definitions/Preferences"
         },
-        "shareType": {
-          "description": "Type of the share.",
+        "deviceType": {
+          "description": "Device type to be used for the job.",
           "enum": [
-            "UnknownType",
-            "HCS",
-            "BlockBlob",
-            "PageBlob",
-            "AzureFile",
-            "ManagedDisk"
+            "DataBox",
+            "DataBoxDisk",
+            "DataBoxHeavy"
           ],
           "type": "string",
-          "readOnly": true,
           "x-ms-enum": {
-            "name": "ShareDestinationFormatType",
+            "name": "SkuName",
             "modelAsExtensible": true,
             "values": [
               {
-                "value": "UnknownType",
+                "value": "DataBox",
+                "description": "Data Box."
+              },
+              {
+                "value": "DataBoxDisk",
+                "description": "Data Box Disk."
+              },
+              {
+                "value": "DataBoxHeavy",
+                "description": "Data Box Heavy."
+              }
+            ]
+          }
+        }
+      },
+      "x-ms-discriminator-value": "ValidatePreferences"
+    },
+    "PreferencesValidationResponseProperties": {
+      "description": "Properties of data center and transport preference validation response.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ValidationInputResponse"
+        }
+      ],
+      "properties": {
+        "status": {
+          "description": "Validation status of requested data center and transport.",
+          "enum": [
+            "Valid",
+            "Invalid",
+            "Skipped"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ValidationStatus",
+            "modelAsExtensible": true,
+            "values": [
+              {
+                "value": "Valid",
+                "description": "Valid"
```

Comment:
this description will have to be improvbed

Reply:
sure